### DISCUSSION TITLE
Props events for Tooltip, Popover

### DISCUSSION THREAD

**Discussion by evgenygorchakov** [unovue/reka-ui#939] [General]
Hello, thanks for the work you have done, the idea of this library is great.

How to use tooltips on mobile devices ? What was your idea to use them on mobile and desktop devices ? I might be missing something.

I need to display a tooltip. The tooltip component is great for this, but it turns out it doesn't work on mobile devices. Then took popover, but on desktop it only opens on click, and I wanted to give users a better experience and show the tooltip on hover.

Of course I've solved this problem already, but it would be great if you would give the option to pass events for this like you do in [floating-vue](https://floating-vue.starpad.dev/guide/config#default-values) (look at the props triggers: ['hover', 'focus', 'touch']).

In the discord community I was advised to accept the fact that there is no hover and make a component that will dynamically display a popup on mobile and a tooltip on desktop. But agree that this is a very strange solution that requires a lot of stuff, as if it could be avoided. I realize that you developed Tooltip this way because you were advised to do so by sites like [nng](https://www.nngroup.com/articles/tooltip-guidelines/) and the like. But let's be reasonable, and recognize that this is all very strange and it's making people suffer. Thank you.

If you're okay with that idea, I might be able to help with this one.

**Comment by nicooprat** [comment]
For anyone hitting this in 2026, we had to add the `disableClosingTrigger` prop so we can show the tooltips on touch devices by clicking on it.

**Reply by evgenygorchakov** [reply]
Thank you, I didn't think anyone would respond.
